src/WeekView/WeekView.js

```diff
@@ -265,47 +267,50 @@ export default class WeekView extends Component {
 
     return (
       <View style={styles.container}>
-        <View
-          style={
-            numberOfDays !== 1
-              ? styles.headerContainer
-              : { opacity: 0, height: 0 }
-          }
-        >
-          <View
-            style={{
-              width: 60,
-            }}
-          />
-          <VirtualizedList
-            horizontal
-            pagingEnabled
-            inverted={horizontalInverted}
-            showsHorizontalScrollIndicator={false}
-            scrollEnabled={false}
-            ref={this.headerRef}
-            data={initialDates}
-            getItem={(data, index) => data[index]}
-            getItemCount={(data) => data.length}
-            getItemLayout={(_, index) => this.getListItemLayout(index)}
-            keyExtractor={(item) => item}
-            initialScrollIndex={this.pageOffset}
-            renderItem={({ item }) => {
-              return (
-                <View key={item} style={styles.header}>
-                  <Header
-                    style={headerStyle}
-                    textStyle={headerTextStyle}
-                    formatDate={formatDateHeader}
-                    initialDate={item}
-                    numberOfDays={numberOfDays}
-                    rightToLeft={rightToLeft}
-                  />
-                </View>
-              );
-            }}
+        {numberOfDays !== 1 ? (
+          <View style={styles.headerContainer}>
+            <View
+              style={{
+                width: 60,
+              }}
+            />
+            <VirtualizedList
+              horizontal
+              pagingEnabled
+              inverted={horizontalInverted}
+              showsHorizontalScrollIndicator={false}
+              scrollEnabled={false}
+              ref={this.headerRef}
+              data={initialDates}
+              getItem={(data, index) => data[index]}
+              getItemCount={(data) => data.length}
+              getItemLayout={(_, index) => this.getListItemLayout(index)}
+              keyExtractor={(item) => item}
+              initialScrollIndex={this.pageOffset}
+              renderItem={({ item }) => {
+                return (
+                  <View key={item} style={styles.header}>
+                    <Header
+                      style={headerStyle}
+                      textStyle={headerTextStyle}
+                      formatDate={formatDateHeader}
+                      initialDate={item}
+                      numberOfDays={numberOfDays}
+                      rightToLeft={rightToLeft}
+                    />
+                  </View>
+                );
+              }}
+            />
+          </View>
+        ) : (
+          <Title
+            style={headerStyle}
+            textStyle={headerTextStyle}
+            selectedDate={selectedDate}
+            format={formatDateHeader}
           />
```

Comment:
Feel free to move the small expression to the top!

Reply:
Lowkey crossed my mind when I wrote this but didn't actually do it lol